client/py/aou_workbench_client/auth.py

```diff
@@ -0,0 +1,82 @@
+"""OAuth against the AoU Workbench API using bearer tokens.
+
+The credentials are obtained by oauth2client, from one of:
+*   Google application default credentials (expected case, from notebook servers).
+*   A private key file, path specified in GOOGLE_APPLICATION_CREDENTIALS environment variable.
+
+For local development using a private key file:
+  api/project.rb get-service-creds --project all-of-us-workbench-test --account $USER@pmi-ops.org
+  GOOGLE_APPLICATION_CREDENTIALS=`pwd`/api/sa-key.json python client/py/aou_workbench_client/auth.py
+"""
+
+import time
+
+from oauth2client.client import GoogleCredentials
+
+from swagger_client.api_client import ApiClient
+
+
+CLIENT_OAUTH_SCOPES = (
+      'https://www.googleapis.com/auth/userinfo.profile',
+      'https://www.googleapis.com/auth/userinfo.email',
+      'https://www.googleapis.com/auth/cloud-billing',
```

Comment:
I don't think we need / want the billing scope here.

Reply:
Removed.